test/bigtest/network/configs/ledgers.js

```diff
@@ -8,6 +8,14 @@ const configLedgers = server => {
   server.get(`${LEDGERS_API}/:id`, (schema, request) => {
     return schema.ledgers.find(request.params.id).attrs;
   });
+
+  server.post(LEDGERS_API, (schema, request) => {
+    const attrs = JSON.parse(request.requestBody) || {};
+
+    return schema.ledgers.create(attrs).attrs;
+  });
+
+  server.put(`${LEDGERS_API}/:id`, () => null);
```

Comment:
what's about `import { createPut } from '@folio/stripes-acq-components/test/bigtest/network/configs';` ?
```suggestion
  server.put(`${LEDGERS_API}/:id`, createPut('ledgers'));
```